<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_mp_potion">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>mp 포션</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_mp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{
	background-color: rgb(187, 187, 187);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	border-style: none;
}
QProgressBar::chunk{
	background-color: rgb(0, 85, 255);
	margin: 3px;
	border-radius: 5px;
}</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>MP %p</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_player_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgb(255, 203, 5);
	color: rgb(53, 100, 173);
	border-color: rgb(53, 100, 173);
	font: 10pt &quot;휴먼매직체&quot;;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_hp_potion">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>hp 포션</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_fight">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>400</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 203, 5);
	color: rgb(53, 100, 173);
	border-color: rgb(53, 100, 173);
	border-radius: 10px;
	font: 12pt &quot;휴먼매직체&quot;;
}</string>
    </property>
    <property name="text">
     <string>전투</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_player_image">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>플레이어 이미지</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_hp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{
	background-color: rgb(187, 187, 187);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	border-style: none;
}
QProgressBar::chunk{
	background-color: rgb(255, 0, 0);
	margin: 3px;
	border-radius: 5px;
}</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>HP %p</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pokemon_image">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>포켓몬 이미지</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pokemon_name">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgb(255, 203, 5);
	color: rgb(53, 100, 173);
	border-color: rgb(53, 100, 173);
	font: 10pt &quot;휴먼매직체&quot;;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_ex">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{
	background-color: rgb(187, 187, 187);
	color: rgb(255, 255, 255);
	border-radius: 5px;
	border-style: none;
}
QProgressBar::chunk{
	
	background-color: rgb(85, 170, 0);
	margin: 3px;
	border-radius: 5px;
}</string>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>EX %p</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgb(255, 203, 5);
	color: rgb(53, 100, 173);
	border-color: rgb(53, 100, 173);
	font: 10pt &quot;휴먼매직체&quot;;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>메소</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
